slave-maven/test/run

```diff
@@ -8,5 +8,6 @@
 
 docker run ${IMAGE_NAME} oc
 docker run --entrypoint="/bin/bash" ${IMAGE_NAME} "-c" "mvn --version"
+docker run --entrypoint="/bin/bash" ${IMAGE_NAME} "-c" "gradle --version"
```

Comment:
it'll break until we have a gradle package for rhel, yeah.  it'd be nice if we could address that, but the alternative is to only run this particular test when the imagename doesn't contain rhel, for now.